src/patterns-hmrc-govuk-team/forms-detailed-guide/index.njk

```diff
@@ -40,6 +45,13 @@ Types of form that can be included in this format are:
 More than one type of form can be hosted on the same page. You can still only use the green button once on the page (check the <a href="/patterns-hmrc-govuk-team/green-button/" class="govuk-link">green button pattern</a> for more information). For example: <a href="https://www.gov.uk/guidance/register-as-a-bingo-promoter" class="govuk-link">Register as a bingo promoter</a>.
 </p>
 
+<h3 class="govuk-heading-m" id="Detailed examples">Detailed guidance examples</h3>
+<ul class="govuk-list govuk-list--bullet">
+  <li><a href="/guidance/register-as-a-bingo-promoter" class="govuk-link">Register as a bingo promoter</a> - an example of an online service with a green button and LC form to register by print and post</li>
```

Comment:
`/guidance/register-as-a-bingo-promoter` which resolves to https://www.tax.service.gov.uk/guidance/register-as-a-bingo-promoter` doesn't exist guessing this should be `gov.uk`?
```suggestion
  <li><a href="https://www.gov.uk/guidance/register-as-a-bingo-promoter" class="govuk-link">Register as a bingo promoter</a> - an example of an online service with a green button and LC form to register by print and post</li>
```